### DISCUSSION TITLE
Compatibilité Linux

### DISCUSSION THREAD

**Discussion by Cardiacman13** [morhue/FF7SYWLinuxInstaller#29] [Ideas]
Bonjour,

Je suis un grand fan du SYW et je fais partie d'une commue Linux Gaming, on rêverait de l'avoir facilement sur Linux. 

Tu penses qu'il y a beaucoup de travail pour l'adapter ?

**Comment by satsukiyatoshi** [comment]
a priori l'installation de la dernière version (unified) passe bien avec lutris : https://ff7.fr/forum/index.php?page=post&ids=473662&pg=1
certes c'est axé steam deck mais comme lutris n'y est pas exclusif, ça doit bien s'adapter ^^
